docs/porting/target/tickless/tickless.md

```diff
@@ -0,0 +1,39 @@
+# Tickless mode
+
+Tickless mode is an optimization mechanism available in RTOS for suspending the SysTick. You can use it in situations when RTOS is idle for multiple ticks, so you can achieve power savings by entering uninterrupted sleep. 
+
+## Reminder on scheduling & sleep modes in Mbed OS
+
+Mbed OS uses the SysTick timer at period of 1ms to process threads' scheduling.
+For instance, a system running two threads would see its timing diagram look like :
+
+![](./resources/Normal_Tick.png)
+
+Note how the device never enters deepsleep and wastes cycles in Systick while all thread are asleep.
```

Comment:
```suggestion
Note that the device never enters deep sleep and wastes cycles in SysTick while all threads are asleep.
```

Reply:
Are we trying to say "Note that the device doesn't enters deep sleep and doesn't waste cycles in SysTick if all threads are asleep"?